deploy/scripts/deploy.ps1

```diff
@@ -70,6 +70,16 @@
 
  .PARAMETER numberOfWindowsGateways
     Number of Windows gateways to deploy into the simulation.
+
+ .PARAMETER gatewayVMSKU
```

Comment:
@saramonteiro, let's make name of the variable came-case to align with the rest. Something like `gatewayVmSku`.